videointelligence/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include README.rst LICENSE requirements.txt
+recursive-include google *.json *.proto
```

Comment:
Compared to the `.proto` file, the generated files are worthless to read for an understanding of the back end surface.  (Actually, they are worthless to read for almost any purpose).